packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -0,0 +1,172 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import { StartEnd } from "../patterns/start-end";
+import { applyMixins } from "../utilities/apply-mixins";
+
+/**
+ * An Option Custom HTML Element.
+ * Implements {@link https://www.w3.org/TR/wai-aria-1.1/#option | ARIA menuitem }.
+ *
+ * @public
+ */
```

Comment:
Does this need proxy logic? I don't think it is form-associated is it?

Reply:
The listbox-option isn't form-associated but it does hold its own `option` element to make option manipulation with the form-associated select significantly easier.

Reply:
gotcha okay